lib/sla.js

```diff
@@ -126,6 +179,7 @@ function originateChannel(client, inbound, bridge, trunkData) {
           unanswered.channel));
       hangup();
     });
+    updateState(client, bridge.name, 'INUSE');
```

Comment:
This should be changed as per Tyler's BLF pull request when rebasing if not already part of his pull request.